Wrappers/Python/cil/optimisation/operators/GradientOperator.py

```diff
@@ -326,8 +327,10 @@ def __init__(self, domain_geometry,  bnd_cond = NEUMANN, **kwargs):
         
         super(Gradient_C, self).__init__(domain_geometry=domain_geometry, 
                                          range_geometry=range_geometry) 
+
```

Comment:
please remove this empty line and the next one. Basically this file remains untouched.